helm_deploy/values-production.yaml

```diff
@@ -33,24 +33,24 @@ service:
   internalPort: 3000
 
 ingress:
-  enabled: true
-  namespace: laa-crime-application-store-production
-  className: modsec
-  annotations:
-    nginx.ingress.kubernetes.io/enable-modsecurity: "true"
-    nginx.ingress.kubernetes.io/modsecurity-snippet: |
-      SecRuleEngine On
-      SecAction "id:900200,phase:1,nolog,pass,t:none,setvar:tx.allowed_methods=GET HEAD POST OPTIONS PUT PATCH DELETE"
-    external-dns.alpha.kubernetes.io/set-identifier: laa-crime-application-store-app-laa-crime-application-store-production-green
-    external-dns.alpha.kubernetes.io/aws-weight: "100"
-  hosts:
-    - host: laa-crime-application-store-prod.apps.live.cloud-platform.service.justice.gov.uk
-      paths:
-        - path: /
-          pathType: Prefix
-  tls:
-    - hosts:
-      - laa-crime-application-store-prod.apps.live.cloud-platform.service.justice.gov.uk
+  enabled: false
+  # namespace: laa-crime-application-store-production
+  # className: modsec
+  # annotations:
+  #   nginx.ingress.kubernetes.io/enable-modsecurity: "true"
+  #   nginx.ingress.kubernetes.io/modsecurity-snippet: |
+  #     SecRuleEngine On
+  #     SecAction "id:900200,phase:1,nolog,pass,t:none,setvar:tx.allowed_methods=GET HEAD POST OPTIONS PUT PATCH DELETE"
+  #   external-dns.alpha.kubernetes.io/set-identifier: laa-crime-application-store-app-laa-crime-application-store-production-green
+  #   external-dns.alpha.kubernetes.io/aws-weight: "100"
+  # hosts:
+  #   - host: laa-crime-application-store-prod.apps.live.cloud-platform.service.justice.gov.uk
+  #     paths:
+  #       - path: /
+  #         pathType: Prefix
+  # tls:
+  #   - hosts:
+  #     - laa-crime-application-store-prod.apps.live.cloud-platform.service.justice.gov.uk
```

Comment:
do we want to keep this or can we just delete those lines?

Reply:
also.. should we also replicate this in UAT as that way we can test any changes that might be related to this (unlikely to be any I know) before relaese

Reply:
yeah I was initially going to keep it in just so we had an example of what we had in production if we need to re-enable but I think as long as we have one (Dev) we can get rid

regarding doing this in UAT also, I agree - I was going to do it in another PR but will do it in this one